packages/@ngtools/logger/src/console-logger-stack.spec.ts

```diff
@@ -1,13 +1,13 @@
 import {LogEntry, Logger} from './logger';
 import {ConsoleLoggerStack} from './console-logger-stack';
 import {NullLogger} from './null-logger';
+import {toArray} from 'rxjs/operators';
```

Comment:
This should be an import directly from 'rxjs/operators/toArray'

Reply:
no, importing from `'rxjs/operators'` is fine

Reply:
This is to maintain parity with angular/material2, angular/flex-layout, and angular/preboot, starting with https://github.com/angular/material2/pull/8160. It seems like it's an open question though on that thread, and will probably become irrelevant as RxJS 6 comes closer

Reply:
Importing from operators does cause a larger bundle currently but from memory that's a bug at the moment and should be fixed. With that in mind I think the more ergonomic approach should be the desired kne

Reply:
Please note also that this is a node application not a web application.  Bundle size is irrelevant.

Reply:
I think it's more about setting a precedent for best practices, as a lot of developers look to first-party packages like this for guidance (node app or no), especially when it comes to RxJS. Again, for this at least it seems to be a non-starter, the only reason I'm leaving this comment up is to archive the discussion.

Reply:
Best practices for RxJS 6 is to `import {toArray} from 'rxjs'`. I'm personally okay with this as it's likely the whole Angular ecosystem will move towards that over the next quarter or two.